modules/apps/portal-security/portal-security-content-security-policy/src/main/java/com/liferay/portal/security/content/security/policy/internal/servlet/taglib/ContentSecurityPolicyTopHeadDynamicInclude.java

```diff
@@ -48,7 +53,16 @@ public void include(
 		printWriter.print(
 			" type=\"text/javascript\">window.Liferay = window.Liferay || ");
 		printWriter.print("{}; window.Liferay.CSP = {nonce: '");
-		printWriter.print(nonce);
+
```

Comment:
do we want to emit an empty, but non undefined value for Liferay.CSP object in this case?

Reply:
Yes. An empty nonce means no nonce. We are not using `null` (neither in the client, nor in the server).